docs/root/intro/arch_overview/security/ssl.rst

```diff
@@ -117,8 +117,8 @@ Certificate selection
 certificates. These may be a mix of RSA and P-256 ECDSA certificates for multiple services. The following rules apply:
 
 Certificate config/loading rules:
-
-* Only one certificate of a particular type (RSA or ECDSA) may be specified.
+* DNS SANs or Subject Common Name is extracted as server name pattern to match SNI during handshake.
+* Only one certificate of a particular type (RSA or ECDSA) may be specified for each server name pattern.
```

Comment:
You'll need to clarify the behavior around wildcards. Is it valid or invalid to have a cert for `test.example.com` and also one for `*.example.com`? If both are valid, please specify which cert is chosen when SNI is `test.example.com`.

Reply:
Got it. Done.